source/open-definition-revision-process-dev.markdown

```diff
@@ -2,19 +2,21 @@
 Definition Revision Approval Process _- draft_
 ====================================
 
-1. Issues with the current Open Definition are identified.  The Open Definition community will disucss on the mailing list and decide which issues are to be dealt with in the next release reach consensus on each of these issues.
+1. Issues with the current Open Definition are identified.  The Open Definition community will disucss on the mailing list or discuss.okfn.org forum and decide which issues are to be dealt with in the next release.
 
 2. A new draft document is started.  Chair notifies the list and other relevant fora that a new draft document has been started summarizing the issues that the upcoming release is to address and inviting participation.
 
-3. Issues with the current Open Definition are identified.  The Open Definition community will discuss on the mailing list and decide which issues are to be dealt with in the next release reach consensus on each of these issues.  The draft document is updated to relfect current consensus.
+3. Along with discussion, people submit pull requests to the draft until it seems to relfect current consensus.
 
-4. The Open Definition Advisory Council chair will summarize the consensus to the Advisory Council on the mailing list and to other relevant fora, okfn-discuss at a minimum.
+4. When no outstanding issues remain unsettled, the Open Definition Advisory Council chair will summarize the consensus to the Advisory Council on the mailing list and to other relevant fora, okfn-discuss at a minimum.
 
-5. If over next two weeks issues are raised which indicate a different consensus or further discussion needed, chair will step process back to step 2 or 3 as appropriate.  If no further issues are raised, the chair notifies the list and other relevant fora that the draft is ready for voting, establishing a release candidate, and informing contributors that only typo corrections will be accepted from this point forward.
+5. If, over next two weeks, issues are raised which indicate a different consensus or further discussion needed, chair will step process back to step 2 or 3 as appropriate.  If no further issues are raised, the chair holds a vote on the OD list to accept the current draft as a release candidate.
```

Comment:
Or, combine this with above and jump *right* to a vote for an RC even. RC could simply mean "the council itself doesn't have any outstanding concerns, so this *migtht* be final depending on wider feedback.